sycl/plugins/opencl/pi_opencl.cpp

```diff
@@ -454,6 +492,322 @@ pi_result OCL(piEnqueueMemBufferMap)(
   return ret_err;
 }
 
+//
+// USM
+//
+
+// Allocates host memory accessible by the device.
+pi_result OCL(piHostMemAlloc)(void **result_ptr, pi_context context,
```

Comment:
Please change these OCL names to follow real PI interface names pextUSM...

Reply:
Ok